src/App.tsx

```diff
@@ -21,6 +35,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={(e) => {
+              setPerPage(Number(e.target.value));
+              setCurrentPage(1);
+            }}
```

Comment:
Pls move the function inside `onChange` handler to a separate const and use as a callback here